### PR TITLE
add automm hpo tutorial

### PR DESCRIPTION
**Pull request by suzhoum**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/tutorials/multimodal/advanced_topics/index.rst [hunk 1]
```diff
@@ -15,10 +15,17 @@ Advanced Topics
...
+      How to customize AutoMM configurations.
+   
 
 .. toctree::
    :maxdepth: 1
    :hidden:
 
    efficient_finetuning_basic
    customization
+   hyperparameter_optimization
```
Comment (sxjscience) [inline docs/tutorials/multimodal/advanced_topics/index.rst:31]: Do you think we should put it before customization ?

Comment (suzhoum) [reply docs/tutorials/multimodal/advanced_topics/index.rst:31]: I can move it, but I did mention `customization` in this tutorial to help readers on the search space.

Comment (sxjscience) [reply docs/tutorials/multimodal/advanced_topics/index.rst:31]: Or shall we move customization to be the first tutorial?

Comment (bryanyzhu) [reply docs/tutorials/multimodal/advanced_topics/index.rst:31]: Customization should be the last tutorial. Every tutorial can link to customization, and use it as a lookup table.

**Review by bryanyzhu — APPROVED** [review]
LGTM.

File: docs/tutorials/multimodal/advanced_topics/index.rst (hunk 1 above)
Comment (suzhoum) [reply docs/tutorials/multimodal/advanced_topics/index.rst]: Sounds good. I moved it before customization.

**Review by sxjscience — APPROVED** [review]

**Pull request merged**